tests/e2e/test_main.py

```diff
@@ -287,6 +295,7 @@ def test_image_build_env(cli_runner: CLIRunner) -> None:
     assert result.returncode == 0, result
 
     secret = str(uuid.uuid4())
+    print("secret", secret)
```

Comment:
do we need it for debugging purposes? 
mb let's hide it behind
```
try:
         assert result.returncode == 0, result
except AssertionError as e:
         click.echo(f"secret: {secret}")
         raise
```
(the same for line 333)

Also, shouldn't the image name in line 165, 271 and 318 be also unique?

Reply:
temp changes too

Reply:
> mb let's hide it behind

no need to add logic for debug purposes only, tests should be as simple as possible